### ISSUE TITLE
Quickbuild not handling subpages

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/library#531]
Noticed this in #493 , where the readiness.ptx file is being changed.  The quick build fails (although a full build succeeds), with the following error:

```
* PTX:ERROR:  the element with the @xml:id given as a subtree root ("VE-intro") is not division that can be chunked into HTML page(s).  Quitting...
```

I believe it is complaining because the root of this file is an `<introduction>` that is included into the `main.ptx` page.  So perhaps what needs to happen is that any file with an `<xi:include/>` checks the included files for changes, and rebuilds if they are changed; and files whose root element is not a `<chapter>` or `<section>` or perhaps a few other limited exceptions are not built in the preview.

**Comment by StevenClontz** [comment]
Working on this at #528

**Comment by siwelwerd** [comment]
Was this fixed by #528?

**Comment by StevenClontz** [comment]
Yep!
